lib/index.js

```diff
@@ -227,6 +227,22 @@ Mongoose.prototype.disconnect = function(fn) {
   });
 };
 
+/**
+ * Getter/setter around function for pluralizing collection names.
+ *
+ * @param {Function|null} [fn] overwrites the function used to pluralize connection names
+ * @return {Function|null} the current function used to pluralize connection names, `undefined` by default.
+ * @api public
+ */
+
+Mongoose.prototype.pluralize = function(fn) {
+  if (arguments.length > 0) {
+    this._pluralize = fn;
+    return fn;
+  }
+  return this._pluralize;
```

Comment:
the `return fn` is unnecessary because you've already set `this._pluralize` to `fn` so you can just use 1 return at the end.